packages/terra-popup/src/_TetherComponent.jsx

```diff
@@ -0,0 +1,251 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tether from 'tether';
+
+const ATTACHMENT_POSITIONS = [
+  'top left',
+  'top center',
+  'top right',
+  'middle left',
+  'middle center',
+  'middle right',
+  'bottom left',
+  'bottom center',
+  'bottom right',
+];
+
+const propTypes = {
+  /**
+   * A hash of tether classes which should be changed or disabled.
+   */
+  classes: PropTypes.object,
+  /**
+   * The prefix placed at the beginning of the default classes.
+   */
+  classPrefix: PropTypes.string,
+  /**
+   * Rule set to pass to tether, contraining the content to by setting 'to', 'attachment', and/or 'pin'.
```

Comment:
Perhaps we should provide a link to the tether docs here? This is a very lengthy comment which corresponds to a variable doing multiple things.